index.md

```diff
@@ -92,10 +94,30 @@ DRAM/PMEM ratio period measured in milliseconds
 
 *memory-ratio-check-period 100*
 
-**Common parameters:**
+### Persistent Memory Variant
+
+When the Memory Allocation Policy is set to a value other than *dram-only* it means that some memory will be allocated on one or more PMEM nodes.
+There are two options for how the Memory Allocator will choose which PMEM NUMA node should be used first and what happens when that node runs out of free space.
+
+Persistent Memory Variant is defined in redis.conf or via command line argument:
+
+* *single* - in this variant memory comes from the closest Persistent Memory NUMA node at the time of allocation. If there is not enough free memory in this node, to satisfy an allocation request, inactive pages from this node are moved into the swap space.
+* *multiple* - in this variant memory comes from the closest Persistent Memory NUMA node available at the time of allocation. When there is not enough free memory in this node, to satisfy an allocation request, the allocation pattern switches to the next closest persistent NUMA node. When available space is exhausted in all persistent NUMA nodes - the swap space is used.
+
+In both variants, if there is no free memory left in the swap space, an "Out of Memory" error occurs. By default TieredMemDB uses a "single" variant.
+
+Example:
+
+Use all available PMEM NUMA nodes
+
+*pmem-variant multiple*
```

Comment:
```suggestion
    ```
    pmem-variant multiple
    ```
```

The "```" quotes are used to denote a code block in github-flavoured markdown.

Please remove leading whitespace from suggestion - leading spaces show a non-formatted block, otherwise they would be treated as markdown by github.

Suggestion, after removing leading whitespace would be formatted as:

```
pmem-variant multiple
```

Reply:
Please check lines 129-138 for correctly formatted code block.